api/android/api/build.gradle

```diff
@@ -60,6 +60,18 @@ android {
     }
     productFlavors {
     }
+    sourceSets {
+        main {
+            if (project.hasProperty('SNPE_DSP_LIBRARY_PATH')) {
+                jniLibs.srcDirs += project.properties['SNPE_DSP_LIBRARY_PATH']
+                println 'Set jniLibs.srcDirs includes libraries for SNPE DSP runtime'
+            }
+        }
+    }
+    packagingOptions {
+        // do not strip libraries for SNPE DSP runtime
+        doNotStrip "*/arm64-v8a/*_skel.so"
```

Comment:
IMHO, these so files are already stripped.
However, to fix the error (or warning) of `no sections` when building Android Studio, 
this code is good.

Reply:
You are correct. I think the previous comments was misleading. I modified the comment, Thanks!